src/include/verify_build.h

```diff
@@ -60,6 +60,13 @@ __wt_verify_build(void)
     WT_SIZE_CHECK(WT_UPDATE, WT_ALIGN(WT_UPDATE_SIZE, 8));
     WT_STATIC_ASSERT(offsetof(WT_UPDATE, data) == WT_UPDATE_SIZE);
 
+    /*
+     * WT_UPDATE: Verify constants used in determining the number of bytes to allocate when there is
+     * no value in the update structure.
```

Comment:
I think it's worth leaving a note here about what's likely to be necessary if the content of the WT_UPDATE structure changes, to help us make the right choice in the future if that change happens.

Reply:
Advisory note added.